src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -56,22 +56,27 @@ def _delete_bid(self, payload):
         try:
             arguments = json.loads(payload["data"])
             assert set(arguments.keys()) == {'bid'}
```

Comment:
Actually what I meant before is that this clause should not be needed anymore. The endpoint should be able to receive an empty dict as argument (even without including the 'bid' member)

Reply:
Ok, ATM i set the bid=None in the delete_bid method in the api_client. 
What you mean is that stetting a bit when wanting to delete all bids is not logical, that is why you want to be able to send delete_bid without any arguments, right?